src/redux/actions/auth.js

```diff
@@ -6,13 +6,13 @@ import { saveToken, removeToken } from '../../utils/localStorage';
 import { getProfileRequest } from './profile';
 
 
-const baseUrl = 'https://vidar-ah-backend-production.herokuapp.com';
-
+// const baseUrl = 'https://vidar-ah-backend-production.herokuapp.com';
```

Comment:
@Dubby20 We shouldn't leave commented out code in the codebase.

Reply:
True it was a mistake. I commented it out when I was testing with localhost. I've removed it now.